euphony/src/androidTest/java/co/euphony/rx/EuRxManagerTest.kt

```diff
@@ -0,0 +1,18 @@
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import co.euphony.rx.EuRxManager
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class EuRxManagerTest {
+
+    @Test
+    public fun testListen() {
+
+        @Test(expected = Test.None::class) /* no exception expected */
+        fun withExceptionHandlingNoErrorOccurs() {
+            val rxManager = EuRxManager()
+            rxManager.listen()
+        }
+    }
```

Comment:
`@Test` is nested. please write only one function 
### option1
```kotlin
        @Test(expected = Test.None::class) /* no exception expected */
        fun withExceptionHandlingNoErrorOccurs() {
            val rxManager = EuRxManager()
            rxManager.listen()
        }
```

### option2
```kotlin
        @Test(expected = Test.None::class) /* no exception expected */
        fun testListen() {
            val rxManager = EuRxManager()
            rxManager.listen()
        }
```

Reply:
I made a ridiculous mistake!!!!!!
I'll fix it